include/tins/pdu_cacher.h

```diff
@@ -53,7 +53,7 @@ namespace Tins {
  * etc.
  */
 template <typename T>
-class PDUCacher : public PDU {
+class TINS_API PDUCacher : public PDU {
```

Comment:
I pushed it yesterday (commit f88d94c
<https://github.com/einarjon/libtins/commit/f88d94cbaa73d6eb4246fbcd2e7e173666253486>
).

It seems to be part of the pull request already, but this is my first
github contribution so I have no idea what I'm doing.

Thanks,
Einar
.

On 17 February 2016 at 18:52, Matias Fontanini <notifications@github.com>
wrote:

> In include/tins/pdu_cacher.h
> <https://github.com/mfontanini/libtins/pull/129#discussion_r53202904>:
>
> > @@ -53,7 +53,7 @@ namespace Tins {
> >   * etc.
> >   */
> >  template <typename T>
> > -class PDUCacher : public PDU {
> > +class TINS_API PDUCacher : public PDU {
>
> Can you remove it? I'll merge it after you do that.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mfontanini/libtins/pull/129/files#r53202904>.
>



-- 
Regards
Einar Jón
+43 650 291 3472